ibm/ibm-how-to/launch-ubuntu-instances.rst

```diff
@@ -0,0 +1,140 @@
+Launch an Ubuntu Virtual Server Instance (VSI) on IBM Cloud VPC infrastructure
+==============================================================================
+
+This how-to guide will give instructions for launching a Virtual Server Instance (VSI) on IBM Cloud using Ubuntu.
+Ensure you have an active IBM Cloud account before starting.
```

Comment:
Link to IBM cloud sign-up page? or to other docs with details about getting an IBM cloud account?